apps/log/lib/log/model/log.ex

```diff
@@ -61,7 +61,7 @@ defmodule Helix.Log.Model.Log do
     %__MODULE__{}
     |> cast(params, @creation_fields)
     |> validate_required(@required_fields)
-    |> put_primary_key()
+    |> put_change(:log_id, PK.pk_for(__MODULE__))
```

Comment:
All models should use autogen. There's no reason for you to try to assume the PK of something before it is inserted into the database as it just doesn't exist